content/django-vs-flask.md

```diff
@@ -0,0 +1,73 @@
+# Django vs Flask: which one should you choose for your project?
+Django and Flask are among the most popular Python web frameworks.
+In this article we're going to review their main strengths and weaknesses to help you decide which one is the better fit for your project.
+
+
+# Django
+
+## Approach
+Django opts for an all batteries included approach.
+This means that we have everything we need to build our project included into the framework.
+from a powerful ORM to an authentication system or a templating engine.
```

Comment:
Not sure, but I think this may be missing a `,` and also adding capital to `From.` Maybe something like this? `From a powerful ORM to an authentication system or a templating engine, Django takes care of it all without the need to learn any extra libraries`.